README.md

```diff
@@ -1,5 +1,109 @@
+  considered to be in the audit-logging project's scope.
++ Submit a ticket for your issue, assuming one does not already exist.
+  + Clearly describe the issue, including steps to reproduce when it is a bug.
+  + Make sure you fill in the earliest version that you know has the issue.
++ Fork the [repository][project-repo] on GitHub, take a look at the [project's JIRA page][project-jira] and [Wiki]([audit-log-wiki]).
+
+Getting Started: Setup
+----------------------
+
+#### TODO - how to setup/start the plugin with Jenkins, etc
+
+
+
+Usage
+----------------------
+
+#### TODO - demo, screenshot, gifs of running audit logging, etc
+
+
+Contributing to the Audit Log Project
+--------------------------
+
+We accept Pull Requests via GitHub. The [gitter](https://gitter.im/jenkinsci/outreachy) is the
+main channel of communication for contributors.
+
+Before you dig right into the code, there are a few guidelines that we need
+contributors to follow so that we can have a chance of keeping on top of
+things and it makes applying PRs easier for us:
++ Create a topic branch from where you want to base your work (this is usually the master branch).
+  Push your changes to a topic branch in your fork of the repository.
++ Make commits of logical units.
++ Respect the original code style: by using the same [codestyle][code-style],
+  patches should only highlight the actual difference, not being disturbed by any formatting issues:
+  + Only use spaces for indentation.
+  + Create minimal diffs - disable on save actions like reformat source code or organize imports.
+    If you feel the source code should be reformatted, create a separate PR for this change.
+  + Check for unnecessary whitespace with `git diff --check` before committing.
++ Make sure your commit messages are in the proper format. Your commit message should contain the key of the JIRA issue.
+```
+[MCOMPILER-XXX] - Subject of the JIRA Ticket
+ Optional supplemental description.
+```
++ Make sure you have added the necessary tests (JUnit/IT) for your changes.
++ Run all the tests with `mvn -Prun-its verify` to assure nothing else was accidentally broken.
++ Submit a pull request to the repository in the Apache organization.
++ Update your JIRA ticket and include a link to the pull request in the ticket.
++ For changes of a trivial nature to comments and documentation, it is not always necessary to
+create a new ticket in JIRA.  In this case, it is appropriate to start the first line of a commit
+with '(doc)' instead of a ticket number.
+
+
+
+Additional Resources
+--------------------
+
++ [General GitHub documentation](https://help.github.com/)
++ [Developing Jenkins Plugins](https://wiki.jenkins.io/display/JENKINS/Extend+Jenkins)
++ [Jenkins Developer Documentation](https://jenkins.io/doc/developer)
++ [Jenkins Main site][jenkins-site]
++ [Jenkins Developer Mailing List](https://jenkins.io/mailing-lists/)
+
+
+[license]: https://todothis.com
```

Comment:
This should be the same link as, well, every other reference to the MIT license. See pom.xml for the canonical link.

Reply:
should i put in any copyright text ?